### ISSUE TITLE
Attachment links not open

### ISSUE THREAD

**Issue by czwi** [leanote/leanote#306]
- using leanote 1.0.5 on Mac OS X and Windows with local database and file management
- when I creating notes (normal or markdown), upload an attachment and make a link nothing happens by clicking the link afterwards
- the attachment is correct stored in the folder for attachments

**Comment by lealife** [comment]
You can download attachment from attachment upload panel

**Comment by czwi** [comment]
Yes, this is possible, but not comfortable. I think it should be possible to click the link and open the standard application with this attachment.
